### ISSUE TITLE
XYDataSeries UpdateSeries

### ISSUE THREAD

**Issue by mike-bower** [NRTDP/SciChartBlazor#12]
Great package, thanks for putting this out there!

I would like to know if it is possible to implement the DataSeries.Update() method? It seems that this feature is missing, and this is unfortunate because it does not allow high speed functionality when displaying real time data.

I may be able to contribute to this project once I get more used to the source code, but right now I am looking for a quick fix, as I have a demo next week

**Comment by mikehollas123** [comment]
Thanks! It's just a little side project I have with my work at Northwestern University. I don't really have a lot of time to quickly add this functionality, but my understanding is that it's mostly abusing the trick of clearing and updating the series rather than starting from scratch. The problem with blazor is that you have to Marshall data between it and JS, which can be slow. We do have an Unmarshalled update function that should hopefully be more performant. Check in the SciChartBuilder class. That should reduce the bottleneck between blazor and JS and likely matches the functionality of the Update() JS method. 
Hope this helps

**Comment by mike-bower** [comment]
Thanks for the quick reply.

I'm still a bit of novice with SciChart, and do most of my work using Blazor, (keeping JS to a minimum).

I'll try the UnmarshalledUpdate method and see if it meets my requirements.

Once I get past my next sprint, I'll try to make some time to contribute to the project, if you are willing to allow me to join the team.

I'm really surprised that the SciChart folks haven't built their own Blazor Wrapper... Last time I reached out to them they basically said to stick to TypeScript...
(Maybe it's all about the Marshalling...)

**Comment by mikehollas123** [comment]
Of course! Please feel free to add a PR, there's no way I can keep up with the SciChart dev team. Back when I started this the SciChart guys were too busy with the JS to really do anything with blazor. They did have a demo, but it was very limited. Things are changing pretty rapidly over there so I've not been able to keep up (most of the changes aren't that useful for our projects so updating them hasn't been a priority). 
I took a closer look, and the update function updates a single y value for a given x index. Right now I don't have anything that could do that, but should be reasonably easy to implement. If you're doing that a lot (with real time data) it might be slow to translate for every point.
